src/main/java/com/tenx/ms/retail/order/rest/dto/OrderStatusEnum.java

```diff
@@ -0,0 +1,35 @@
+package com.tenx.ms.retail.order.rest.dto;
+
+import com.tenx.ms.commons.rest.BaseValueNameEnum;
+import io.swagger.annotations.ApiModel;
+
+@ApiModel(value = "Order Status")
+public enum OrderStatusEnum implements BaseValueNameEnum<OrderStatusEnum> {
+    ORDERED(1, "ORDERED"),
+    PACKING(2, "PACKING"),
+    SHIPPED(3, "SHIPPED"),
+    INVALID(4, "INVALID");
+
+    private int value;
```

Comment:
@marielbysoares , @alfredo-github 
   On this you should use either the Codeable or the BaseValueNameEnum classes, so that you can use it both on the Entity and on the DTO.

   I think that for simplicity sake you can use the Codeable enum.

   Also bear in mind that if you do that you need to specify a Converter for the Entity.

   An example can be seen on: 
Enum: http://git.prod.auction.local/projects/HS/repos/homes-ms-search/browse/src/main/java/com/tenx/ms/homes/search/rest/dto/enums/SearchFrequencyEnum.java?at=refs%2Fheads%2Fdevelop

Converter: http://git.prod.auction.local/projects/HS/repos/homes-ms-search/browse/src/main/java/com/tenx/ms/homes/search/util/SearchFrequencyConverter.java?at=refs%2Fheads%2Fdevelop

Entity usage:
http://git.prod.auction.local/projects/HS/repos/homes-ms-search/browse/src/main/java/com/tenx/ms/homes/search/domain/SaveSearchEntity.java?at=refs%2Fheads%2Fdevelop